seaborn/categorical.py

```diff
@@ -2663,21 +2645,22 @@ def boxenplot(
     {saturation}
     {width}
     {dodge}
-    k_depth : "proportion" | "tukey" | "trustworthy", optional
+    k_depth : "proportion", "tukey", "trustworthy" or scalar, optional
```

Comment:
I think the official numpydoc standard is to enclose options in curly braces. You need to double up the curly braces to not mess with the string formatting.

Reply:
Thanks, will do this.